src/components/PlayerScoreBoard/index.scss

```diff
@@ -0,0 +1,67 @@
+@import '../../styles/variables';
+
+.playerscoreboard {
+  align-items: center;
+  display: flex;
+
+  flex-direction: column;
+  font-size: 1.4em;
+
+  justify-content: flex-start;
+
+  list-style-type: none;
+
+  padding: 0;
+
+  color: $text-light;
+}
+
+.playerscore {
+  align-items: stretch;
+
+  box-sizing: border-box;
+  border-radius: 0;
+
+  display: flex;
+
+  justify-content: space-between;
+  max-width: 10em;
+
+  padding: 0.7rem 1.6rem;
+
+  text-align: center;
+
+  width: 100%;
+
+  & ~ & {
+    border-top: $linen solid 2px;
+    margin-top: 0;
+  }
+
+  // First place
+  &:nth-child(1) {
+    background-color: $linen;
+    color: $text-dark;
+  }
+
+  // Second place
+  &:nth-child(2) {
+  }
+
+  // Third place
+  &:nth-child(3) {
+  //  background-color: $white-snow;
+  //  color: $text-dark;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/Strooped/strooped-master/pullRequest?prid=5419925)